src/App.js

```diff
@@ -1,7 +1,35 @@
 import "./App.css";
-
+import { Routes, Route } from "react-router-dom";
+import { Header, Sidebar } from "./components";
+import {
+	Home,
+	VideoListing,
+	PlayList,
+	WatchLater,
+	History,
+	LikedVideos,
+} from "./pages";
```

Comment:
Can export all from index.js and then import here from 1 file

Reply:
Yes. I am exporting from common index.js. like from "./pages/index.js'